lesson19/js/components/contentsUI.js

```diff
@@ -0,0 +1,153 @@
+import { differenceInDays } from "date-fns";
+import { createAttributedElements } from "../utils/createAttributedElements";
+
+export const renderTabs = ({ data }, parent) => {
+  const articleTabContents = createAttributedElements("div", { class: "tab" });
+  articleTabContents.appendChild(tabContents.getTabContents(data));
+  parent.appendChild(articleTabContents);
+};
+
+const switchTabs = (e) => {
+  const eventTarget = e.target;
+  if (eventTarget === e.currentTarget) return;
+  const targetPanel = eventTarget.getAttribute("aria-controls");
+  const activeTab = document.querySelector('[aria-selected="true"]');
+  const activePanel = document.querySelector('[aria-hidden="false"]');
+  activeTab.setAttribute("aria-selected", "false");
+  eventTarget.setAttribute("aria-selected", "true");
+  activePanel.setAttribute("aria-hidden", "true");
```

Comment:
nits
If you don't have to support IE, I thought this could be written in this way.
```suggestion
  activeTab.ariaSelected = "false";
  eventTarget.ariaSelected = "true";
  activePanel.ariaHidden = "true";
```

Reply:
What's that!!!!　
Looks amazing.

I didn't know that. Thanks for letting me know!
Correction.
[09ea050](https://github.com/443n0511/js-lesson/pull/41/commits/09ea050dc2c3566f4829297f7d813128ebcde420)